File: C++/bee 1042.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

using namespace std;

void bubbleSort(int arr[], int len) {
    len--;
    for (int i = 0; i < len; i++) {
        for (int j = 0; j < len; j++) {
            if (arr[j] > arr[j+1]) {
                int aux = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = aux;
            }
        }
    }
}

int main() {

    int a, b, c, aux;
    cin >> a >> b >> c;

    int numeros[] = {a,b,c};
    bubbleSort(numeros, 3);


    cout << numeros[0] << endl << numeros[1] << endl << numeros[2] << endl;
    cout << endl << a << endl << b << endl << c << endl;


        return 0;
}

File: Java/bee 1042.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a, b, c;
        a = sc.nextInt();
        b = sc.nextInt();
        c = sc.nextInt();

        int[] arr = new int[]{a,b,c};
        arr = bubbleSort(arr);
        System.out.println(arr[0] + "\n" + arr[1] + "\n" + arr[2] + "\n");
        System.out.println(a + "\n" + b + "\n" + c);

    }

    public static int[] bubbleSort(int[] arr){
        for(int i = 0; i < arr.length - 1; i++){
            for(int j = 0; j < arr.length - 1; j++){
                if(arr[j] > arr[j+1]) {
                    int aux = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = aux;
                }
            }
        }
        return arr;
    }
}
